### PR TITLE
Add changelog entry for 3.0.9

### PR DESCRIPTION
**Pull request by blink1073**
## References
Update changelog for upcoming 3.0.9 release.

## Code changes
None

## User-facing changes
Updated changelog.

## Backwards-incompatible changes
None

### CODE REVIEW

File: docs/source/getting_started/changelog.rst [hunk 1]
```diff
@@ -10,6 +10,15 @@ See the `JupyterLab
 3.0 <https://github.com/jupyterlab/jupyterlab/milestone/48?closed=1>`__
 milestone on GitHub for the full list of pull requests and issues closed.
 
+v3.0.9
+^^^^^^
+* Remove the previous ``file_to_run`` logic. [#9848](https://github.com/jupyterlab/jupyterlab/pull/9848)
```
Comment (jasongrout) [inline docs/source/getting_started/changelog.rst:15]: This is still an RST file in 3.0.x :)

Comment (blink1073) [reply docs/source/getting_started/changelog.rst:15]: Hahaha whoops!

**Comment by jasongrout** [comment]
Looks good. The link check failure is expected since master has moved the changelog file.

**Comment by blink1073** [comment]
Release in progress

**Comment by blink1073** [comment]
https://pypi.org/project/jupyterlab/3.0.9/

**Comment by jasongrout** [comment]
@meeseeksdev please backport to master

**Pull request merged**